account_invoice_production_lot/models/account_invoice.py

```diff
@@ -35,17 +36,38 @@ def _compute_prod_lots(self):
         for line in self:
             if not line.order_line_ids:
                 return
-            line.prod_lot_ids = self.mapped(
-                'order_line_ids.procurement_ids.move_ids.lot_ids')
+            line.prod_lot_ids = self.mapped('move_line_ids.lot_ids')
```

Comment:
maybe here you mean `line.mapped...`?

Reply:
Done in https://github.com/OCA/account-invoice-reporting/pull/85/commits/b0f167deba50042fae1060a6e949173d717de247